release/artifacts.yaml

```diff
@@ -237,3 +237,5 @@
   type: jar-scala
 - target: //ledger/error:error
   type: jar-scala
+- target: //ledger/error/generator:lib
```

Comment:
the artifact here seems to be off, CI gets unhappy about it

Reply:
Hmm, that's because we can't generate 2.12 with it. Fabio specialized it to 2.13 only. Can we specify that we only publish for 2.13?

Reply:
not atm, it’s not super hard to fix though.